### PR TITLE
Develop Initial GitHub Webhook Controller tests

### PR DESCRIPTION
**Pull request by johnsgill3**
Put together basic test driver to send WebHook messages without requiring GitHub.

### CODE REVIEW

**Comment by johnsgill3** [comment]
@ryanbmilbourne How about a review?

File: app/controllers/webhook_controller.rb [hunk 1]
```diff
@@ -3,33 +3,36 @@ class WebhookController < ApplicationController
...
 
         # Make sure that this hook came from a known repo
-        verify_signature(request.body.read)
+        verify_signature(@payload_body)
 
         case request.env['HTTP_X_GITHUB_EVENT']
             when "pull_request"
                 if @payload["action"] == "opened"
-                    puts "Received: pull_request[opened]"
+                    puts "WHC Received: pull_request[opened]"
```
Comment (ryanbmilbourne) [inline app/controllers/webhook_controller.rb]: Can we add timestamps for these?  Would help with log readibility.

Comment (johnsgill3) [inline app/controllers/webhook_controller.rb]: Good :point_up:. I'll add that now,

**Comment by ryanbmilbourne** [comment]
:+1: Yay UTC

**Pull request merged**